### PR TITLE
Feature/bare metal form

### PR DESCRIPTION
**Pull request by temesghentekeste**
Greetings from Temesghen, 

Initially, I created a repository by the name forms-rails. As I have followed git-flow, I created a feature branch called bare-metal-form, and using that branch, I developed the project. Bootstrap is used is to style the views and kaminari gem to paginate the users listing page.

Kindly request a pull request before I merge my feature branch to the develop branch.

Regards,
Temesghen T.

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @temesghentekeste 
### You have done well so far :clap: However, you have a few issues to fix:-
- [ ] Update the routes, see below resources :users, only: [:new, :create, :edit, :update]
- [ ] Remove the comments in your codebase
### Make all the changes and submit for another code review. Good luck.
- Happy coding!

**Comment by temesghentekeste** [comment]
Thanks, @rOluochKe for your valuable feedback. I have added the requirements you have suggested and kindly request a new pull request before I merge my feature branch to the develop branch.

Thanks in advance.

File: app/views/shared/_errors.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+<% if obj.errors.any? %>
+  <div class="row errors">
+    <div class="col-sm-10 mx-auto">
+      <div class="card arlert alert-danger p-2 mb-3">
```
Comment (kubilaycaglayan) [inline app/views/shared/_errors.html.erb:5]: :bulb: [OPTIONAL] There shouldn't be any logic(if) in the views. You should move them into the helpers.

**Resources** 
- [Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code)
- [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)
- [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright

**Review by kubilaycaglayan — APPROVED** [review]
Hello Team :wave: 
Status: Project Approved :green_circle: 

Great job team :+1:  :blush: 
Great partial usage! :star: 

:bulb: Please check the inline comments. It is a mandatory requirement for the rest of the Rails section.

With all my respect...

File: app/views/shared/_errors.html.erb (hunk 1 above)
Comment (temesghentekeste) [reply app/views/shared/_errors.html.erb:5]: :pray: Thanks so much Kubilay Caglayan. Your suggestions are helpful, :ok_hand: . I will definitely apply them in my projects. With great courtesy,  keep up the good work. :100: :thumbsup:

**Pull request merged**